indexer/services/ender/src/handlers/stateful-order/conditional-order-triggered-handler.ts

```diff
@@ -39,6 +40,8 @@ export class ConditionalOrderTriggeredHandler extends
 
     const indexerOrder: IndexerOrder = orderTranslations.convertToIndexerOrderWithSubaccount(
       order, perpetualMarket, subaccount);
+
+    await sendOrderTriggeredNotification(order, perpetualMarket, subaccount);
```

Comment:
could be also good to test for this e2e in `services/ender/__tests__/lib/on-message.test.ts`